packages/ui/src/Avatar/AvatarGroupExpanded.tsx

```diff
@@ -0,0 +1,33 @@
+import { Avatar } from './Avatar'
+
+import type { AuthorExtended } from '@filecoin-foundation/utils/types/authorTypes'
+
+type AuthorWithCompany = Omit<AuthorExtended, 'bio'>
+
+export type AvatarGroupExpandedProps = {
+  authors: Array<AuthorWithCompany>
+}
+
+export function AvatarGroupExpanded({ authors }: AvatarGroupExpandedProps) {
+  return (
+    <div className="flex flex-col gap-6 sm:flex-row sm:items-center sm:gap-10">
```

Comment:
Wondering if we could use `flex-wrap`? That way, in a case if there's, let's say 3 or more authors, or if someone has a really long company title, we don't have to worry about breaking the layout...